lib/features/add_property/view/widgets/add_property_type_view.dart

```diff
@@ -132,9 +133,7 @@ class _PropertyTypeCardView extends StatelessWidget {
       : super(key: key);
 
   @override
-  Widget build(BuildContext context) => AspectRatio(
```

Comment:
based on the design, width and height are not the same. Using AspectRatio with ratio = 1 will break the UI in case the text is too long and displayed in 2 lines